### PR TITLE
Add CircleCI Cmd

### PR DESCRIPTION
**Pull request by shopetan**
## What
Add CircleCI Cmd in biko.

## Why
We need to find if we develop some software.

### CODE REVIEW

File: cli/circleci.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			newCircleCIJobsCmd(),
+		},
+	}
+}
+
+func newCircleCIJobsCmd() cli.Command {
+	return cli.Command{
+		Name:    "jobs",
+		Aliases: []string{"j"},
+		Usage:   "Open incident page",
```
Comment (KeisukeYamashita) [inline cli/circleci.go]:
```suggestion
		Usage:   "Open jobs page",
```

File: providers/circleci/circleci.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+// Init ...
+func (p *Provider) Init(c *cli.Context) error {
+	p.Ctx = c
+	return nil
+}
+
+// GetTargetURL ...
+func (p *Provider) GetTargetURL() (string, error) {
+	var baseURL = fmt.Sprintf("https://circleci.com/gh/%s/", p.Ctx.String("org"))
```
Comment (KeisukeYamashita) [inline providers/circleci/circleci.go]:
```suggestion
	var baseURL = fmt.Sprintf("https://circleci.com/gh/%s/", p.Org)
```

**Comment by KeisukeYamashita** [comment]
For alias, see https://github.com/KeisukeYamashita/biko/blob/53174206541a0b460a927b72c42b4d2ebdd26b7f/providers/youtube/youtube.go#L63-L83.

These methods are common but implemented in all provider.  I will refactor these methods to functions later, but please use alias for now like other providers. Thanks:)

**Comment by KeisukeYamashita** [comment]
@shopetan  Don't forget to update the README.md too. I appreciate your work.

File: providers/circleci/circleci.go (hunk 2 above)
Comment (shopetan) [reply providers/circleci/circleci.go]: If use `subcommand` , Can not set variable of `p.Org`. So I delete `p.Org` and use `p.Ctx.String("org")`. Do you have any idea?

**Comment by KeisukeYamashita** [comment]
@shopetan  I am sorry I fixed the pagerduty too. https://github.com/KeisukeYamashita/biko/pull/38

File: providers/circleci/circleci.go (hunk 2 above)
Comment (KeisukeYamashita) [reply providers/circleci/circleci.go]: @shopetan Sorry for the PagerDuty example. You can do like this. 
https://github.com/KeisukeYamashita/biko/blob/dc2849c5ccd84ea410ffbf1af06ebcc8ddc33aa7/cli/pagerduty.go#L52-L59

As same to the `$ biko cc` command. Implement in the struct.

**Pull request merged**